### PR TITLE
Ncsdk 5370 release notes v1.3.0

### PR DESCRIPTION
**Pull request by ru-fu**
PDF printout (links don't work):
[nRF Connect SDK v1.3.0 Release Notes.pdf](https://github.com/nrfconnect/sdk-nrf/files/4771743/nRF.Connect.SDK.v1.3.0.Release.Notes.pdf)
Ping me for the link to the HTML output.

Please check the question marks - they usually indicate that the link might be wrong (for example, pointing to the sample but should point to the library).

### CODE REVIEW

File: doc/release-notes-1.3.0.rst [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+
+* Added:
+
+  * :ref:`lib_date_time` library - maintains the current date-time UTC from multiple time sources (modem, NTP servers).
+  * :ref:`lib_ftp_client` library - can be used to download or upload FTP server files.
+  * :ref:`lib_nrf_cloud_agps` library - provides a way to request and process A-GPS data from nRF Cloud to be used with the nRF9160 SiP.
+    The :ref:`nrf_cloud_agps_sample` sample demonstrates how to use the library.
+  * :ref:`connectivity_bridge` application with dual UART and Bluetooth LE support - replaces the :ref:`usb_uart_bridge_sample` sample.
+
+* :ref:`gps_with_supl_support_sample` sample: (?)
```
Comment (shantha-14) [inline doc/release-notes-1.3.0.rst:62]: https://github.com/nrfconnect/sdk-nrf/tree/master/samples/nrf9160/gps

Comment (ru-fu) [reply doc/release-notes-1.3.0.rst:62]: That is where the link currently points.

**Comment by kl-cruz** [comment]
There is lack of CMakeLists.txt file when develops application on the nRF52833 that emulates nRF52820. This file must be added here: `https://github.com/zephyrproject-rtos/zephyr/tree/master/boards/arm/nrf52833dk_nrf52820` and must contain:

`
zephyr_compile_definitions(DEVELOP_IN_NRF52833)
zephyr_compile_definitions(NRFX_COREDEP_DELAY_US_LOOP_CYCLES=3)
`
This file will be included automatically by the build system. There is no additional point to do.

see similar file: https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf52840dk_nrf52811/CMakeLists.txt

cc: @barsok @anangl

File: doc/release-notes-1.3.0.rst [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+
+Thread
+------
+
+Added the following samples:
+
+* :ref:`ot_cli_sample` sample - very basic sample with Thread enabled and shell support to allow using the OpenThread Command Line Interface.
+  This sample is needed for Thread certification.
+* :ref:`coap_client_sample` sample and :ref:`coap_server_sample` sample - two samples demonstrating how to use the CoAP protocol over a Thread network.
+* Thread CoAP Multiprotocol Bluetooth LE Client sample - the multiprotocol extension of the :ref:`coap_client_sample` sample.
```
Comment (MarekPorwisz) [inline doc/release-notes-1.3.0.rst:177]: There is no multiprotocol support as the driver support was removed.

**Comment by ru-fu** [comment]
> 
> 
> There is lack of CMakeLists.txt file when develops application on the nRF52833 that emulates nRF52820. This file must be added here: `https://github.com/zephyrproject-rtos/zephyr/tree/master/boards/arm/nrf52833dk_nrf52820` and must contain:
> 
> `zephyr_compile_definitions(DEVELOP_IN_NRF52833) zephyr_compile_definitions(NRFX_COREDEP_DELAY_US_LOOP_CYCLES=3)`
> This file will be included automatically by the build system. There is no additional point to do.
> 
> see similar file: https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf52840dk_nrf52811/CMakeLists.txt
> 
> cc: @barsok @anangl

Where should this go? Known issues > Build system? Or Known issues > BLE? Or Known issues > nRF52820?
Is there a Jira issue that we should mention?

File: doc/release-notes-1.3.0.rst (hunk 2 above)
Comment (ru-fu) [reply doc/release-notes-1.3.0.rst:177]: Removed

File: doc/release-notes-1.3.0.rst [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+
+* The Antenna Diversity feature is not supported on nRF52840 devices. (KRKNWK-6361)
+
+Thread
+------
+
+* It is not possible to build Thread samples using SEGGER Embedded Studio (SES).
+  SES does not support .cpp files in |NCS| projects. (NCSDK-5014)
+* It is not possible to provision the :ref:`coap_client_sample` sample to servers that it cannot directly communicate with.
+  This is because Link Local Address is used for communication. (KRKNWK-6358) (?)
```
Comment (MarekPorwisz) [inline doc/release-notes-1.3.0.rst:478]: not sure what this question mark is for. If it is about issue being closed here is the still open issue: KRKNWK-6427

Comment (ru-fu) [reply doc/release-notes-1.3.0.rst:478]: This was the original input:

> It is not possible to provision CoAP Client with servers it is not able to communicate directly as Link Local Address is used for communication (KRKNWK-6358).

I wasn't sure if my interpretation of what communicates with what here is correct. ;)

Comment (MarekPorwisz) [reply doc/release-notes-1.3.0.rst:478]: ok. Looks good.

File: doc/release-notes-1.3.0.rst [hunk 4]
```diff
@@ -0,0 +1,571 @@
...
+* :ref:`nrfxlib:nfc` - added documentation about :ref:`nrfxlib:type_2_tag` and :ref:`nrfxlib:type_4_tag`, updated the :ref:`nrfxlib:nfc_integration_notes`
+* :ref:`nrfxlib:nrf_security_readme` - updated
+
+Known issues
+************
+
+nRF9160
+=======
+
+* The :ref:`asset_tracker` applications prints warnings and error messages during successful FOTA. (NCSDK-5574)
```
Comment (umapraseeda) [inline doc/release-notes-1.3.0.rst:443]:
```suggestion
* The :ref:`asset_tracker` application prints warnings and error messages during successful FOTA. (NCSDK-5574)
```

File: doc/release-notes-1.3.0.rst [hunk 5]
```diff
@@ -0,0 +1,571 @@
...
+
+* On nRF5340, only the :ref:`nrfxlib:ble_controller` is supported for Bluetooth Mesh. (NCSDK-5580)
+
+Common
+======
+
+Samples and applications
+------------------------
+
+* The build configuration :ref:`bootloader`, :ref:`secure_partition_manager`, and application does not work.
```
Comment (umapraseeda) [inline doc/release-notes-1.3.0.rst:498]:
```suggestion
* The build configuration :ref:`bootloader`, :ref:`secure_partition_manager`, and application do not work.
```

Comment (ru-fu) [reply doc/release-notes-1.3.0.rst]: No, it's actually "the build configuration consisting of ..." that does not work. I was wondering about that sentence as well - thanks for confirming it's hard to understand. ;) Tried to fix it now.

File: doc/release-notes-1.3.0.rst [hunk 6]
```diff
@@ -0,0 +1,569 @@
...
+
+Thread
+------
+
+* It is not possible to build Thread samples using SEGGER Embedded Studio (SES).
+  SES does not support .cpp files in |NCS| projects. (NCSDK-5014)
+* It is not possible to provision the :ref:`coap_client_sample` sample to servers that it cannot directly communicate with.
+  This is because Link Local Address is used for communication. (KRKNWK-6358)
+* The "diag" command is not yet supported by Thread in the |NCS|. (KRKNWK-6408)
+
```
Comment (wbober) [inline doc/release-notes-1.3.0.rst:478]:
```
Zigbee
-------
* There may be a noticeable delay (~220 ms) between calling ZBOSS API and on-the-air activity.

```

Comment (mariuszpos) [reply doc/release-notes-1.3.0.rst:478]: * ZBOSS alarms are inaccurate. On average they last longer by 6.4%. It is recommended to use Zephyr alarms.

Comment (ru-fu) [reply doc/release-notes-1.3.0.rst:478]: Added.

File: doc/release-notes-1.3.0.rst [hunk 7]
```diff
@@ -0,0 +1,577 @@
...
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.0 for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+
+Highlights
+**********
+
+* Support for Thread and Zigbee
```
Comment (pdunaj) [inline doc/release-notes-1.3.0.rst:17]:
```
nRF Desktop released
```

Comment (pdunaj) [reply doc/release-notes-1.3.0.rst:17]: @jhn-nordic

Comment (jhn-nordic) [reply doc/release-notes-1.3.0.rst:17]: "nRF Desktop released" looks good to me.

Comment (MonikaLieLarsen) [reply doc/release-notes-1.3.0.rst:17]: "released" sounds a bit confusing in this context (everything in release notes is released...). "Support for nRF Desktop" or "nRFDesktop supported for production" or "nRFDesktop supported for development" is good - take your pick, @pdunaj :)

**Comment by MonikaLieLarsen** [comment]
> Where should this go? Known issues > Build system? Or Known issues > BLE? Or Known issues > nRF52820?
> Is there a Jira issue that we should mention?

I'd also like to see a jira ticket number, if we plan to do something about this?
Suggest we put it under Known issues > nRF52820 as it is so specific.

**Comment by shantha-14** [comment]
> > There is lack of CMakeLists.txt file when develops application on the nRF52833 that emulates nRF52820. This file must be added here: `https://github.com/zephyrproject-rtos/zephyr/tree/master/boards/arm/nrf52833dk_nrf52820` and must contain:
> > `zephyr_compile_definitions(DEVELOP_IN_NRF52833) zephyr_compile_definitions(NRFX_COREDEP_DELAY_US_LOOP_CYCLES=3)`
> > This file will be included automatically by the build system. There is no additional point to do.
> > see similar file: https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf52840dk_nrf52811/CMakeLists.txt
> > cc: @barsok @anangl
> 
> Where should this go? Known issues > Build system? Or Known issues > BLE? Or Known issues > nRF52820?
> Is there a Jira issue that we should mention?

This should go to Build System https://github.com/zephyrproject-rtos/zephyr/pull/26181 PR upstream would have fixed the issue

File: doc/release-notes-1.3.0.rst [hunk 8]
```diff
@@ -0,0 +1,577 @@
...
+  * Added APN readout.
+
+* :ref:`lte_lc_readme` library:
+
+  * Added the concept of events.
+  * Allowed to modify PSM/eDRX parameters at runtime.
+  * Allowed to modify PDP context and PDN authentication at runtime.
+
+* :file:`lib/at_host` library: (?)
+
```
Comment (jhn-nordic) [inline doc/release-notes-1.3.0.rst:103]: remove (?)

lib/at_host does not have documentation it seems.

Comment (ru-fu) [reply doc/release-notes-1.3.0.rst:103]: Right, that's why we cannot link. But should the changes be removed? Or do the changes actually belong to a different library?

Comment (ru-fu) [reply doc/release-notes-1.3.0.rst:103]: ok

File: doc/release-notes-1.3.0.rst [hunk 9]
```diff
@@ -0,0 +1,577 @@
...
+    The :ref:`nrf_cloud_agps_sample` sample demonstrates how to use the library.
+  * :ref:`connectivity_bridge` application with dual UART and Bluetooth LE support - replaces the :ref:`usb_uart_bridge_sample` sample.
+
+* :ref:`gps_with_supl_support_sample` sample:
+
+  * Reworked the API to make it more flexible.
+  * Added an option to register an event handler.
+    All event types from the GPS will be received, as opposed to the previous single-trigger source.
+
+* :ref:`lib_nrf_cloud` library: (?)
```
Comment (MonikaLieLarsen) [inline doc/release-notes-1.3.0.rst:68]: Link OK

File: doc/release-notes-1.3.0.rst [hunk 10]
```diff
@@ -0,0 +1,577 @@
...
+
+  * Added FOTA download progress monitoring.
+  * Added :ref:`lib_nrf_cloud_agps` support. (?)
```
Comment (MonikaLieLarsen) [inline doc/release-notes-1.3.0.rst:71]: Link OK

File: doc/release-notes-1.3.0.rst [hunk 11]
```diff
@@ -0,0 +1,577 @@
...
+  * Added support for MQTT persistent sessions.
+  * Moved cloud connection polling out of the application into a separate nrf_cloud thread.
+  * Exposed MQTT topics to the nRF Cloud library.
+
+* :ref:`lib_aws_fota` library: (?)
```
Comment (MonikaLieLarsen) [inline doc/release-notes-1.3.0.rst:76]: Link OK

File: doc/release-notes-1.3.0.rst [hunk 12]
```diff
@@ -0,0 +1,577 @@
...
+
+* :ref:`supl_client`:
+
+  * Provided version 0.6.0 for download.
+    This new version is required for compatibility with |NCS| v1.3.0.
+
+nRF5
+====
+
+The following changes are relevant for the nRF52 and nRF53 Series. (?)
```
Comment (MonikaLieLarsen) [inline doc/release-notes-1.3.0.rst:153]: Yes, remove question mark?

**Comment by ru-fu** [comment]
Updated with the requested changes.
All changes since the original version are currently in a separate commit (to make it easier to check). I'll squash that commit before we merge.

File: doc/release-notes-1.3.0.rst [hunk 13]
```diff
@@ -0,0 +1,593 @@
...
+  There is support for a hardware rollback counter that must be provided as part of the platform, as well as a software solution that protects against some types of rollback.
+* Added an optional boot record in shared memory to communicate boot attributes to code that is run later.
+* Added a cleanup of the Arm core before the application is booted.
+* Allowed recovery over USB CDC ACM with logs enabled.
+* Various fixes to work with the latest Zephyr version.
+
+Build system
+============
+
+* Added support for external flash in the :ref:`secure_partition_manager` sample. (? or is this a duplicate of the last bullet?)
```
Comment (MonikaLieLarsen) [inline doc/release-notes-1.3.0.rst:388]: @carlescufi did you add the last bullet point on purpose, or is it a duplicate?

File: doc/release-notes-1.3.0.rst [hunk 14]
```diff
@@ -0,0 +1,593 @@
...
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Renamed the subsystem to ``esb`` and moved it from :file:`subsys/enhanced_shockburst` to :file:`subsys/esb`.
+* Renamed the header file and all data types to align with the name change.
+
+Common
+======
+
```
Comment (MonikaLieLarsen) [inline doc/release-notes-1.3.0.rst:265]: Do we need a "The following changes are relevant for all device families" here?

File: doc/release-notes-1.3.0.rst [hunk 15]
```diff
@@ -0,0 +1,598 @@
...
+
+Highlights
+**********
+
+* Support for Thread and Zigbee
+* Support for Bluetooth Mesh
+* nRF Desktop supported for production
+* New nRF9160 samples and libraries
+* Application update through external flash
```
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/release-notes-1.3.0.rst:21]: Updated (without the "added" to be more consistent with the other highlights).

File: doc/release-notes-1.3.0.rst [hunk 16]
```diff
@@ -0,0 +1,598 @@
...
+  A delay of 1000 ms is required for this to work as intended.
+
+nRF5
+====
+
+nRF5340
+-------
+
+* FOTA with the :ref:`zephyr:smp_svr_sample` does not work.
+
```
Comment (shantha-14) [inline doc/release-notes-1.3.0.rst:470]: Bluetooth "peripheral_hids_keyboard" does not work as intended for nRF53 target. @grochu  agree ?

File: doc/release-notes-1.3.0.rst (hunk 13 above)
Comment (carlescufi) [reply doc/release-notes-1.3.0.rst:393]: This is a duplicate, please remove

Comment (ru-fu) [reply doc/release-notes-1.3.0.rst:393]: Removing Secure Partition Manager and leaving Partition Manager in.

File: doc/release-notes-1.3.0.rst (hunk 16 above)
Comment (shantha-14) [reply doc/release-notes-1.3.0.rst:470]: discussed with @grochu  nothing to be added here

**Pull request merged**